### PR TITLE
Mark SI test runs as unstable on cluster launch failure.

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
We want the system integration tests to be marked as unstable and not as
failed if the cluster launch failed. This gives us a more fine grained
overview of the daily runs.

### CODE REVIEW

File: ci/launch_cluster.sh [hunk 1]
```diff
@@ -62,11 +62,11 @@ function create-junit-xml {
 
 if ! ./dcos-launch create; then
   create-junit-xml "dcos-launch" "cluster.create" "Cluster launch failed."
-  exit 1
+  exit 0
```
Comment (jeschkies) [inline ci/launch_cluster.sh]: If a build returns 0 but JUnit finds and error it will report the build as unstable.

Comment (zen-dog) [reply ci/launch_cluster.sh]: Oh yes, this should be a comment in code. Otherwise the next guy will wonder.

Comment (ichernetsky) [reply ci/launch_cluster.sh]: 👍

**Comment by jeschkies** [comment]
Ok, it works now https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/system-integration-tests/job/marathon-si-dcos-open/job/karsten%252Fcluster-launch-unstable//

File: ci/launch_cluster.sh [hunk 2]
```diff
@@ -44,29 +44,11 @@ template_parameters:
...
-	      </testcase>
-	  </testsuite>
-	</testsuites>
-	EOF
-}
-
 if ! ./dcos-launch create; then
-  create-junit-xml "dcos-launch" "cluster.create" "Cluster launch failed."
-  exit 1
+  exit 2
```
Comment (zen-dog) [inline ci/launch_cluster.sh]: should we not comment the possible exit codes at the top?

**Comment by jeschkies** [comment]
The build is green https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/system-integration-tests/job/marathon-si-dcos-open/job/karsten%252Fcluster-launch-unstable/14/

File: ci/launch_cluster.sh (hunk 2 above)
Comment (ichernetsky) [reply ci/launch_cluster.sh]: 👍 , since they are used in a case statement in the other script.

**Pull request merged**